README.md

```diff
@@ -1,7 +1,15 @@
 # honnef.co/go/tools
 
-`honnef.co/go/tools/...` is a collection of tools and libraries for
-working with Go code, including linters and static analysis.
+This is a collection of tools and libraries for working with Go code,
+including linters and static analysis.
```

Comment:
I personally liked the previous version of the paragraph, that mentions the import path pattern, better. Why do you think it's better to remove it?